packages/sources/react/src/components/actions/VtmnLink/VtmnLink.tsx

```diff
@@ -17,6 +17,13 @@ export interface VtmnLinkProps extends React.ComponentPropsWithoutRef<'a'> {
    */
   standalone?: boolean;
 
+  /**
+   * Whether link is reversed or not.
+   * @type {boolean}
+   * @defaultValue false
+   */
+  reversed?: boolean;
```

Comment:
Can you set reversed = false please ? 
It's for the testing library, if an export are not set with a default value, for him, it's a required value